module_utils/network_lsr/argument_validator.py

```diff
@@ -1866,6 +1866,17 @@ def _ipv4_enabled(connection):
             )
             return connection["ip"]["dhcp4"] or has_addrs4
 
+        def _ipv6_ignore(connection):
```

Comment:
What about `_ipv6_is_not_present()`? The current name made me think it will ignore ipv6..

Reply:
how about `_ipv6_is_not_configured()`? Is this better ?

Reply:
Yes, that is good too :-)

Reply:
Updated, thanks.